torch/backends/cudnn/rnn.py

```diff
@@ -35,20 +35,13 @@ def __setstate__(self, state):
         self.inner = None
 
 
-def init_dropout_descriptor(handle, dropout, train, dropout_seed, dropout_state):
+def init_dropout_state(ty, dropout, train, dropout_seed, dropout_state):
     dropout_desc_name = 'desc_' + str(torch.cuda.current_device())
     dropout_p = dropout if train else 0
     if (dropout_desc_name not in dropout_state) or (dropout_state[dropout_desc_name].get() is None):
         dropout_state[dropout_desc_name] = Unserializable(
-            cudnn.DropoutDescriptor(handle, dropout_p, dropout_seed)
+            torch._C._VariableFunctions._cudnn_init_dropout_state(ty, dropout_p, train, dropout_seed)
+            if dropout_p != 0 else None
         )
-    dropout_desc = dropout_state[dropout_desc_name].get()
-    dropout_desc.set_dropout(dropout_p, dropout_seed)
-    return dropout_desc
-
-
-def get_dropout_state(fn, handle):
```

Comment:
we don't need this for backwards compatibility too?

Reply:
I was under the impression that `torch.backends.cudnn.rnn` was private. If it wasn't, I've got a lot more things I've got to bring back...